apps/contrib/schema.py

```diff
@@ -63,17 +64,42 @@ class Meta:
 
 
 class ClientType(DjangoObjectType):
+    use_cases = graphene.List(graphene.NonNull(ClientUseCaseEnum), description="List of use cases for the client.")
+    use_cases_display = EnumDescription(description="Display string for the client's use case.")
+
     class Meta:
         model = Client
         fields = (
             'id',
             'name',
             'is_active',
             'code',
+            'acronym',
+            'contact_name',
+            'contact_email',
+            'contact_website',
             'created_by',
+            'created_at',
+            'other_notes',
+            'opted_out_of_emails',
             'last_modified_by',
+            'modified_at',
         )
 
+    def resolve_use_cases_display(self, info):
+        """
+        Resolve method for use_case_display field. Returns a list of labels for the client's use cases.
+        """
+        try:
+            use_case_labels = [
+                str(Client.USE_CASE_TYPES.__labels__.get(use_case, str(use_case)))
+                for use_case in self.use_cases
+            ]
+        except KeyError as e:
+            use_case_labels = [f"Invalid use case: {e}"]
+
+        return use_case_labels
```

Comment:
This should be fine.

When USE_CASE_TYPES are changed, we need to migrate the data accordingly.

```suggestion
    def resolve_use_cases_display(self, info):
        """
        Resolve method for use_case_display field. Returns a list of labels for the client's use cases.
        """
        return [
            Client.USE_CASE_TYPES(use_case).label
            for use_case in self.use_cases
        ]
```